CMakeLists.txt

```diff
@@ -49,22 +49,30 @@ fix8_setbuildtype(FIX8 Release)
 fix8_addoption("BUILD_UNITTESTS|enable building unit tests|true")
 fix8_addoption("BUILD_ALL_WARNINGS|enable building with all warnings|true")
 
+# ----------------------------------------------------------------------------------------
+# c++ settings
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
 # ----------------------------------------------------------------------------------------
 # deps
-find_package(ZLIB REQUIRED)
+#find_package(ZLIB REQUIRED)
```

Comment:
Um this was working ok , why changed?

Reply:
Required for windows - i'll do that conditional